Assets/Mirror/Runtime/CustomAttributes.cs

```diff
@@ -82,4 +82,17 @@ public class ClientCallbackAttribute : Attribute { }
     /// Converts a string property into a Scene property in the inspector
     /// </summary>
     public class SceneAttribute : PropertyAttribute { }
+
+    /// <summary>
+    /// Used for classes that want to be shown in the NetworkBehaviourInspector 
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, Inherited = true)]
+    public class ShowSyncObjectInInspectorAttribute : Attribute { }
```

Comment:
do we need a new attribute?  why not just show anything that implements SyncObject and IEnumerable?